ffmpeg/ffmpeg.go

```diff
@@ -165,9 +288,13 @@ func HasZeroVideoFrameBytes(data []byte) (bool, error) {
 	vcodec_c := C.CString(strings.Repeat("0", 255))
 	defer C.free(unsafe.Pointer(acodec_c))
 	defer C.free(unsafe.Pointer(vcodec_c))
-	bres := int(C.lpms_get_codec_info(cfname, vcodec_c, acodec_c))
+	var params_c C.codec_info
```

Comment:
Should we free `params_c` struct too? Would be great to avoid duplicating struct initialization snippet here.

Reply:
Fixed